src/main.js

```diff
@@ -151,7 +224,4 @@ function showRandomPoster () {
   // 6 show another random poster
   // 7 make your own poster
   // 8 create poster
-  // 9 get random poster /randomizer   
-  
-  //just went through the assignment and looked at the buttons on the example and in the rubric!//
-  //brainstorming!
\ No newline at end of file
+  
```

Comment:
Great! Glad most of the code worked and only needed minor tweaks. Let's start on iteration 2!